diff --git a/node_modules/sanbase/src/components/Navbar/Search/AssetsCategory.js b/node_modules/sanbase/src/components/Navbar/Search/AssetsCategory.js
index 9073fee..51550a2 100644
--- a/node_modules/sanbase/src/components/Navbar/Search/AssetsCategory.js
+++ b/node_modules/sanbase/src/components/Navbar/Search/AssetsCategory.js
@@ -32,7 +32,7 @@ const useSearchableAssets = (assets) =>
 
 export const propsAccessor = ({ slug }) => ({
   key: slug,
-  to: '/charts?slug=' + slug,
+  to: 'https://app.santiment.net/charts?slug=' + slug
 })
 
 export const Asset = ({ name, ticker }) => (
diff --git a/node_modules/sanbase/src/components/Navbar/Search/Category.js b/node_modules/sanbase/src/components/Navbar/Search/Category.js
index 75f6c21..cdaff49 100644
--- a/node_modules/sanbase/src/components/Navbar/Search/Category.js
+++ b/node_modules/sanbase/src/components/Navbar/Search/Category.js
@@ -1,11 +1,13 @@
 import React, { useEffect } from 'react'
 import cx from 'classnames'
-import { Link } from 'react-router-dom'
 import styles from './Category.module.scss'
 
 // eslint-disable-next-line
 const Title = (props) => <h3 className={styles.title} {...props} />
 
+
+const Link = ({to, ...props}) =>  <a {...props} href={to} target='_blank' rel='noopener noreferrer' />
+
 export const Button = ({ className, As = Link, isCursored, ...props }) => (
   <As {...props} className={cx(styles.button, className, isCursored && styles.button_cursored)} />
 )
diff --git a/node_modules/sanbase/src/components/Navbar/Search/InsightsCategory.js b/node_modules/sanbase/src/components/Navbar/Search/InsightsCategory.js
index 0aef9e4..54e126b 100644
--- a/node_modules/sanbase/src/components/Navbar/Search/InsightsCategory.js
+++ b/node_modules/sanbase/src/components/Navbar/Search/InsightsCategory.js
@@ -1,7 +1,7 @@
 import React, { useState, useEffect, useMemo } from 'react'
 import gql from 'graphql-tag'
 import Category from './Category'
-import { client } from '../../../apollo'
+import { client } from '@src/apollo/client'
 import styles from './Category.module.scss'
 
 const DEFAULT_SUGGESTIONS = []
diff --git a/node_modules/sanbase/src/components/Navbar/Search/PeopleCategory.js b/node_modules/sanbase/src/components/Navbar/Search/PeopleCategory.js
index 4a62d26..434000e 100644
--- a/node_modules/sanbase/src/components/Navbar/Search/PeopleCategory.js
+++ b/node_modules/sanbase/src/components/Navbar/Search/PeopleCategory.js
@@ -18,7 +18,7 @@ const POPULAR_AUTHORS_QUERY = gql`
 
 const propsAccessor = ({ id }) => ({
   key: id,
-  to: '/profile/' + id,
+  to: 'https://app.santiment.net/profile/' + id
 })
 
 function peoplePredicate(value) {
diff --git a/node_modules/sanbase/src/components/Navbar/Search/Suggestions.js b/node_modules/sanbase/src/components/Navbar/Search/Suggestions.js
index d66988a..bfc9e4c 100644
--- a/node_modules/sanbase/src/components/Navbar/Search/Suggestions.js
+++ b/node_modules/sanbase/src/components/Navbar/Search/Suggestions.js
@@ -13,7 +13,7 @@ import styles from './Suggestions.module.scss'
 
 const DEFAULT_RECENTS = []
 
-const Suggestions = ({ suggestionsRef, isOpened, isTablet, isLaptop, ...props }) => {
+const Suggestions = ({ className, suggestionsRef, isOpened, isTablet, isLaptop, ...props }) => {
   const isNotSearched = !props.searchTerm
   const [recents, setRecents] = useState(DEFAULT_RECENTS)
 
@@ -39,9 +39,7 @@ const Suggestions = ({ suggestionsRef, isOpened, isTablet, isLaptop, ...props })
       dropdown.style.minWidth =
         (dropdownWidth > availableWidth ? dropdownWidth : availableWidth) + 'px'
 
-      const isSmallScreen = isTablet || isLaptop
-
-      dropdown.style.right = isSmallScreen ? availableWidth + 40 + 'px' : availableWidth / 2 + 'px'
+      dropdown.style.right = availableWidth / 2 + 'px'
     }
   }, [isOpened])
 
@@ -52,7 +50,7 @@ const Suggestions = ({ suggestionsRef, isOpened, isTablet, isLaptop, ...props })
 
   return (
     <CSSTransition in={isOpened} timeout={500} classNames={styles}>
-      <div ref={suggestionsRef} className={cx(styles.dropdown, styles.exitDone)}>
+      <div ref={suggestionsRef} className={cx(styles.dropdown, styles.exitDone, className)}>
         <RecentsCategory {...props} items={recents} onClear={onRecentsClear} />
         <AssetsCategory {...props} />
         <WalletsCategory {...props} />
diff --git a/node_modules/sanbase/src/components/Navbar/Search/TrendingWordsCategory.js b/node_modules/sanbase/src/components/Navbar/Search/TrendingWordsCategory.js
index 7e283f6..37a7505 100644
--- a/node_modules/sanbase/src/components/Navbar/Search/TrendingWordsCategory.js
+++ b/node_modules/sanbase/src/components/Navbar/Search/TrendingWordsCategory.js
@@ -18,7 +18,7 @@ const TRENDING_WORDS_QUERY = gql`
   }
 `
 
-const TREND_LINK = '/labs/trends/explore/'
+const TREND_LINK = 'https://app.santiment.net/labs/trends/explore/'
 
 const propsAccessor = ({ word, key = word, As }) => ({
   key,
diff --git a/node_modules/sanbase/src/components/Navbar/Search/WalletsCategory.js b/node_modules/sanbase/src/components/Navbar/Search/WalletsCategory.js
index 682f92d..186bea5 100644
--- a/node_modules/sanbase/src/components/Navbar/Search/WalletsCategory.js
+++ b/node_modules/sanbase/src/components/Navbar/Search/WalletsCategory.js
@@ -5,7 +5,7 @@ import styles from './Category.module.scss'
 import { getAddressInfrastructure } from '../../../utils/address'
 
 const DEFAULT_SUGGESTIONS = []
-const WALLET_LINK = '/labs/balance?address='
+const WALLET_LINK = 'https://app.santiment.net/labs/balance?address='
 
 const propsAccessor = ({ word, key = word, As }) => ({
   key,
diff --git a/node_modules/sanbase/src/components/Navbar/Search/index.js b/node_modules/sanbase/src/components/Navbar/Search/index.js
index fce8943..0679807 100644
--- a/node_modules/sanbase/src/components/Navbar/Search/index.js
+++ b/node_modules/sanbase/src/components/Navbar/Search/index.js
@@ -11,7 +11,7 @@ import styles from './index.module.scss'
 
 const EDITABLE_TAGS = new Set(['INPUT', 'TEXTAREA'])
 
-const Search = () => {
+const Search = ({className, dropdownClassName}) => {
   const inputRef = useRef()
   const [isOpened, setIsOpened] = useState(false)
   const [searchTerm, setSearchTerm] = useState('')
@@ -65,7 +65,7 @@ const Search = () => {
 
   return (
     <UISearch
-      className={cx(styles.search, isOpened && styles.search_focused)}
+      className={cx(styles.search, isOpened && styles.search_focused, className)}
       inputClassName={styles.input}
       forwardedRef={inputRef}
       placeholder='Search for assets, trends, etc...'
@@ -80,6 +80,7 @@ const Search = () => {
         searchTerm={searchTerm}
         isOpened={isOpened}
         onSuggestionSelect={onSuggestionSelect}
+        className={dropdownClassName}
       />
     </UISearch>
   )
diff --git a/node_modules/sanbase/src/redux.js b/node_modules/sanbase/src/redux.js
index 903c28e..0ea272b 100644
--- a/node_modules/sanbase/src/redux.js
+++ b/node_modules/sanbase/src/redux.js
@@ -1,21 +1,2 @@
-import { createStore, applyMiddleware } from 'redux'
-import { composeWithDevTools } from 'redux-devtools-extension'
-import { createBrowserHistory } from 'history'
-import { routerMiddleware } from 'react-router-redux'
-import { createEpicMiddleware } from 'redux-observable'
-import { client } from './apollo'
-import epics from './epics/rootEpics.js'
-import reducers from './reducers/rootReducers.js'
-
-export const history = createBrowserHistory()
-
-const middleware = [
-  createEpicMiddleware(epics, {
-    dependencies: {
-      client,
-    },
-  }),
-  routerMiddleware(history),
-]
-
-export const store = createStore(reducers, {}, composeWithDevTools(applyMiddleware(...middleware)))
+export const history = {}
+export const store = {}
